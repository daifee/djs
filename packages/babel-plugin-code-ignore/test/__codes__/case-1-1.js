
// @code-ignore-next-statement kuwo
import kugou from './kugou';

// @code-ignore-next-statement kugou
import kuwo from './kuwo';

function fun() {
  // @code-ignore-next-statement kuwo
  console.log(kugou);
  console.log('---------------');

  console.log(kuwo); // @code-ignore-statement kugou
}

// @code-ignore-next-statement kugou kuwo
function ignore() {
  // 
}

kugou(); // @code-ignore-statement kuwo

kuwo(); // @code-ignore-statement kugou