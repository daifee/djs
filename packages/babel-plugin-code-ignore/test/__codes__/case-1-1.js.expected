"use strict";

var _kugou = _interopRequireDefault(require("./kugou"));
require("./kugou.css");
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
// @code-ignore-next-statement kuwo

// empty

/**
 * css
 * kugou
 */
// @code-ignore-next-statement kuwo

(0, _kugou.default)(); // @code-ignore-statement kuwo

// @code-ignore-statement kugou